queue_services/entity-filer/src/entity_filer/filing_processors/registration.py

```diff
@@ -116,38 +112,43 @@ def process(
     if not (firm_reg_num := legal_entity_info.get_next_corp_num("FM")):
         raise DefaultException(f"registration {filing_rec.id} unable to get a Firm registration number.")
 
-    match legal_type:
-        case LegalEntity.EntityTypes.SOLE_PROP:
+    match entity_type:
+        case BusinessCommon.EntityTypes.SOLE_PROP:
             # Get or create LE
-            business = merge_sp_registration(firm_reg_num, filing, filing_rec)
+            business, alternate_name = merge_sp_registration(firm_reg_num, filing, filing_rec)
 
-        case LegalEntity.EntityTypes.PARTNERSHIP:
+        case BusinessCommon.EntityTypes.PARTNERSHIP:
             # Create Partnership
-            business = merge_partnership_registration(firm_reg_num, filing, filing_rec, registration_filing)
+            business, alternate_name = merge_partnership_registration(
+                firm_reg_num, filing, filing_rec, registration_filing
+            )
 
         case _:
             # Default and failed
             # Based on the above checks, this should never happen
             raise DefaultException(f"registration {filing_rec.id} had no valid Firm type.")
 
     # Assuming we should not reset this from a filing
-    if not business.tax_id:
-        business.tax_id = registration_filing.get("business", {}).get("taxId", None)
+    # if not business.tax_id:
+    #     business.tax_id = registration_filing.get("business", {}).get("taxId", None)
 
-    business.state = LegalEntity.State.ACTIVE
+    business.state = BusinessCommon.State.ACTIVE
+    alternate_name.state = BusinessCommon.State.ACTIVE
 
     if nr_number := business_info_obj.get("nrNumber", None):
         # TODO check how this is getting used, may need operating name?
         filing_meta.registration = {
             **filing_meta.registration,
             **{
                 "nrNumber": nr_number,
-                "legalName": business_info_obj.get("legalName", None),
+                "operatingName": business_info_obj.get("legalName", None),
```

Comment:
Do we need to change `legalName` to `operatingName` here? Or just keep it?

Reply:
From what I can tell, the filing meta data is mainly used by the reports in legal api. I couldn't actually find anywhere where `legalName` is used from the filing meta (except maybe [here for continuation out](https://github.com/bcgov/lear/blob/feature-legal-name/legal-api/src/legal_api/reports/business_document.py#L447)). I think it's okay to leave as `legalName`, especially since this is in reference to the name request,  which [still uses legalName](https://github.com/bcgov/business-schemas/blob/feature-legal-name/src/registry_schemas/schemas/name_request.json#L19).